ffi-cdecl/zeromq_cdecl.c

```diff
@@ -112,7 +119,6 @@ cdecl_func(zhash_cursor)
 cdecl_func(zhash_lookup)
 cdecl_func(zmsg_destroy)
 cdecl_func(zhash_destroy)
-cdecl_func(zframe_destroy)
```

Comment:
It looks like all the destroys are purposefully together. You prefer them as all the zframes together?

Reply:
There is other destroys that are grouped with their own class functions. I'll move the other destroys to their own  groups. So all functions to be grouped by their class (zhash, zctx, zframe etc.)